src/browser/modules/EditorInfo/CreateRelationship.jsx

```diff
@@ -19,11 +34,12 @@ export default class CreateRelationship extends Component {
 
   state = {
     selectedType: null,
-    selectedLabel: null
+    selectedLabel: null,
+    selectedNode: null
   }
 
   render () {
-    const { selectedType, selectedLabel } = this.state
+    const { selectedType, selectedLabel, selectedNode } = this.state
```

Comment:
can we have this component as functional component. use usestate where you want to use state